openapi.yaml

```
@@ -555,6 +555,463 @@ paths:
+                    session_id = response.json()['session_id']
+                    print(f"Session ID: {session_id}")
+                    print(f"Part size: {part_size}")
+                else:
+                    print(f"Error: {response.status_code} - {response.text}")
+
+  /v2/batches/multipart-upload/sessions/{session_id}/parts/{part_num}:
+    put:
+      operationId: uploadMultipartPart
+      x-fern-audiences:
+        - public
+      tags:
+        - batches
+      summary: Upload part to multipart session
+      description: |
+        Upload part of a file to the multipart upload session. Each part's size should match the `part_size` returned by the [Start multipart upload session](/api-sdk/api-reference/batches/create-multipart-upload-session) call, except for the final part, which can be smaller.
+        
+        Parts must be uploaded with consecutive part numbers starting at 1.
+      parameters:
+        - name: session_id
+          in: path
+          required: true
+          schema:
+            type: string
+          description: The session ID obtained from the Start multipart upload session response.
+        - name: part_num
+          in: path
+          required: true
+          schema:
+            type: integer
+          description: The part number, starting at 1 and increasing consecutively for each part.
+        - $ref: '#/components/parameters/ib_context'
+      requestBody:
+        required: true
+        content:
+          application/octet-stream:
+            schema:
+              type: string
+              format: binary
+              description: Raw content of the part to be uploaded.
+      responses:
+        '201':
+          description: The part was successfully uploaded.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/multipartUploadPartResponse'
+        default:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/error'
+          description: Error response.
+      x-fern-examples:
+        - code-samples:
+            - sdk: curl
+              code: |
+                  curl -X PUT "${API_ROOT}/v2/batches/multipart-upload/sessions/<SESSION_ID>/parts/1" \
+                    -H "Authorization: Bearer ${API_TOKEN}" \
+                    -H "IB-Context: ${IB_CONTEXT}" \
+                    -H "Content-Type: application/octet-stream" \
+                    --data-binary "@part1.bin"
+            - sdk: python
+              name: with SDK
+              code: |
+                  # AI Hub Python SDK example
+                  from aihub import AIHub
+
+                  client = AIHub(api_root="<API-ROOT>",
+                                 api_key="<API-TOKEN>",
+                                 ib_context="<IB-CONTEXT>")
+
+                  # Assuming session_id and part_size from previous step
+                  session_id = "<SESSION_ID>"
```

```suggestion
                  session_id = “<SESSION-ID>"
```

Team Docs uses hyphens instead of underscores for all placeholders to distinguish them from Python constants.